### DISCUSSION TITLE
cannot input anything in ubuntu23.04

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [davatorium/rofi#1845] [(Possible) Issues]
when my ubuntu20.04 upgrade to ubuntu23.04 ,rofi cannot get window focus so it cannot input anything

**Comment by DaveDavenport** [comment]
sounds like you are using wayland?

**Reply by DreamMaoMao** [reply]
thanks for your reminder.
yes,ubuntu23.04 is using wayland.
Is there any way to solve this problem？

**Reply by DaveDavenport** [reply]
switch to Xorg or use the lbonn fork of rofi. 
https://github.com/lbonn/rofi/

**Reply by DreamMaoMao** [reply]
I found that there is a button in the lower right corner of the restart login interface to switch back to xorg. The problem is solved, thank you for your answer
